### DISCUSSION TITLE
NUnit.Engine.NUnitEngineException System.IO.FileLoadException : 'nunit.framework, Version=3.13.2.0

### DISCUSSION THREAD

**Discussion by Robin-m-Sartorius** [nunit/nunit#4163] [Q&A]
I am getting: 
```
NUnit.Engine.NUnitEngineException : An exception occurred in the driver while loading tests.
  ----> System.IO.FileLoadException : Could not load file or assembly 'nunit.framework, Version=3.13.2.0, Culture=neutral, PublicKeyToken=2638cd05610744eb' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)

```
 When running my unit tests through a coverage PowrShell script. The tests work from within VisualStudio.
  
 This is a variant on a theme that dates back several versions. It appears that is not fixed, even though some variants of this issue claim it is.

 I do not have a temp folder  as mentioned in
  https://stackoverflow.com/questions/48526149/how-to-fix-could-not-load-file-or-assembly-nunit-engine-version-3-7-0-0

 The project was running nunit tests successfully both in VS and in the coverage script, with no change to Nunit configuration.
 
 I have added new unit tests but it is very unclear why any of those would break NUnit.

I think the critical point is:
"The located assembly's manifest definition does not match the assembly reference."

Could someone give a better description as to what should go in the manifest? And why I would suddenly need a manifest when I did not before?

Or do I need something in the app.config file? In which case some clarity in the error message would be useful. Similarly a description of what is needed would be welcome

I am raising this as a discussion, because it is quite probably caused by user error and it is not an issue.

**Comment by OsirisTerje** [comment]
Sorry, we missed this question.
Did you ever got this resolved?
